apps/editor/src/app/locales/en.json

```diff
@@ -1316,6 +1317,27 @@
         "body": "Do you really want to delete the event \"{{name}}\"?",
         "delete": "Yes, delete the event"
       }
+    },
+    "dashboard": {
+      "dashboard": "Dashboard",
+      "greeting": "Hello, {{name}}!",
+      "yearlySubscribers": "New subscribers over past year",
+      "activity": "Latest Activity",
+      "event": "Event",
+      "summary": "Summary",
+      "noData": "No current subscriber data",
+      "newArticle": "New article created",
+      "newPage": "New page created",
+      "newComment": "New comment added",
+      "newPoll": "New poll opened",
+      "newSubscription": "New subscription",
+      "newAuthor": "New author added",
+      "newUser": "New user added",
+      "newSubscribers": "New subscribers over past year",
+      "createdBy": " by {{creator}}",
+      "subscriberChart": {
+        "fillColor": "#3498ff"
```

Comment:
Color should not be in translation file, but adding it any other way as a string was creating errors.

Reply:
You can add it to the .eslintrc file as an exception to the rule. Shouldn't be in translations I agree.
Else you can also exclude the rule for that line.

Reply:
Thanks I'll do that.